net/tstun/wrap.go

```diff
@@ -1389,6 +1408,37 @@ var (
 	metricPacketOutDropSelfDisco = clientmetric.NewCounter("tstun_out_to_wg_drop_self_disco")
 )
 
+type TrafficAction string
+
+const (
+	TrafficActionAccept    TrafficAction = "accept"
+	TrafficActionDropACL   TrafficAction = "drop_acl"
+	TrafficActionDropError TrafficAction = "drop_error"
+	TrafficActionDropDst   TrafficAction = "drop_dst_unknown"
+)
+
+type trafficLabel struct {
+	// Action indicates what we have done with the packet, and has the following wvalues:
+	// - accept
+	// - drop_acl (rejected packets because of ACL)
+	// - drop_error (rejected packets because of an error)
+	// - drop_dst_unknown
+	Action TrafficAction
+}
+
+var (
+	metricInboundPacketsTotal = usermetrics.NewMultiLabelMap[trafficLabel](
+		"tailscaled_inbound_packets_total",
```

Comment:
Based on some recent discussions, I think we're going to be capturing packet counts by path (direct vs derp) in another package, right? In that case perhaps exporting packet counts here is unnecessary, and we can instead make this package export dropped packets? E.g.

```
tailscaled_inbound_dropped_packets_total{reason="acl"}
tailscaled_outbound_dropped_packets_total{reason="error"}